corehq/apps/linked_domain/util.py

```diff
@@ -96,6 +96,7 @@ def _get_missing_multimedia(app, old_multimedia_ids=None):
             continue
         try:
             local_media = CommCareMultimedia.get(media_info['multimedia_id'])
+            local_media.get_display_file()
```

Comment:
Is this going to slow down the pull since this would load files for each MM?

Reply:
Yes, it will probably take longer.

Reply:
Should this happen only when the media is missing?

Reply:
@sravfeyn , can you elaborate? Multimedia is "stored" in a few places - CommCareMultimedia stores media metadata in couch with a pointer to the actual file in the blobdb.  Line 98 checks if it is missing from couch, and I added line 99 to make sure the actual file exists in the blobdb. So this is basically checking that the media exists.